src/views/users/User/UserCreateUpdate.vue

```diff
@@ -234,8 +238,13 @@ export default {
       // SECURITY_MFA_AUTH 0 不开启 1 全局开启 2 管理员开启
       const adminUserIsNeed = (user?.is_superuser || user?.is_org_admin) && this.$route.meta.action === 'update' &&
         store.getters.publicSettings['SECURITY_MFA_AUTH'] === 2
-      if (store.getters.publicSettings['SECURITY_MFA_AUTH'] === 1 || adminUserIsNeed) {
-        this.fieldsMeta['mfa_level'].disabled = true
+      if (store.getters.publicSettings['SECURITY_MFA_AUTH'] === 1) {
+        this.fieldsMeta['mfa_level'].options = [{ 'value': 3, 'label': this.$t('MFAAllUsers') }]
+        this.initial.mfa_level = 3
+      }
+      if (store.getters.publicSettings['SECURITY_MFA_AUTH'] === 2 && adminUserIsNeed) {
+        this.fieldsMeta['mfa_level'].options = [{ 'value': 4, 'label': this.$t('MFAOnlyAdminUsers') }]
+        this.initial.mfa_level = 4
       }
     }
   }
```

Comment:
This snippet seems to be part of an implementation or configuration file for a system, where it contains methods which manipulate certain object data like IDs and MFA settings. The provided comments are helpful but do not include any specific changes or optimizations that could be suggested here.

As there aren't any obvious issues with the current state of the code, I would suggest looking into security practices and ensuring all necessary updates occur according to best coding standards before publishing this file publicly, especially concerning privacy and compliance aspects. Additionally, making explicit mention during commentaries is crucial in improving code readability and understanding.